requirements.txt

```diff
@@ -52,3 +51,5 @@ geopandas
 
 # TODO: remove threadpoolctl and joblib if not needed anymore for scikit-learn
 # google-cloud-bigquery needs libffi-dev
+
+# scikit-learn
```

Comment:
As this change is not relevant for this PR, please undo, as it changes the history (and especially in these requirement files, a git blame is often very helpful).